scanomatic/ui_server_data/qc_norm.html

```diff
@@ -3,40 +3,19 @@
   <head>
     <title>Quality Control Prototype</title>
     <meta charset="utf-8" />
-    <script src="js/external/jquery.js"></script>
-    <script src="js/external/bootstrap.js"></script>
-    <script src="js/external/d3.js"></script>
-    <script src="js/external/jquery.treetable.js"></script>
-    <script src="js/external/jquery.modal.js"></script>
-    <script src="js/external/jquery-ui.js"></script>
-    <script src="js/external/bootstrap-toggle.js"></script>
-    <script src="js/external/spin.js"></script>
-
-    <script src="js/qc_normHelper.js?ver=1.0.0"></script>
-    <script src="js/qc_normDrawPlate.js?ver=1.0.0"></script>
-    <script src="js/qc_normDrawCurves.js?ver=1.0.0"></script>
-    <script src="js/qc_normAPIHelper.js?ver=1.0.0"></script>
-
+    <script src="js/somlib/som.js?ver=1.0.0"></script>
```

Comment:
why this ver here?

Reply:
You tell me!

Reply:
I have no clue

Reply:
I'll just remove it, as you suggested above then.

Reply:
Fixed, but see comment up top.